edge-patron/src/main/resources/prokopovych/edge-patron/features/patrons.feature

```diff
@@ -61,12 +61,12 @@ Feature: patron tests
     * call read('classpath:prokopovych/edge-patron/features/util/initData.feature@PostOwnerAndFine')
 
     Given url edgeUrl
-    And path 'patron/account/' + extSystemId + '?includeCharges=true'
+    And path 'patron/account/' + extSystemId
     And param apikey = apikey
+    And param query = 'includeLoans=true'
     When method GET
     Then status 200
     And match response.totalChargesCount == 1
-    And match response.charges[0].item.itemId == itemId
```

Comment:
Please revert the code from the scenarios where your change is not there.

Reply:
reverted